docs/src/_getting-started/index.md

```diff
@@ -127,7 +127,7 @@ This will allow you to use [`git diff`](https://git-scm.com/docs/git-diff) to ea
 
 ## View and Activate Your Environments
 
-As part of creating your Meltano project, we automatically added your first [environments](/concepts/environments) called `dev`, `staging` and `prod`. This allows you to define configurations specific to the environment in which you're running your project. Theres also a [`default_environment`](https://docs.meltano.com/concepts/environments#default-environments) setting in the `meltano.yml` that get automatically set to `dev`, you can list and change the active environment using:
+As part of creating your Meltano project, we automatically added your first [environments](/concepts/environments) called `dev`, `staging` and `prod`. This allows you to define configurations specific to the environment in which you're running your project. There's also a [`default_environment`](https://docs.meltano.com/concepts/environments#default-environments) setting in the `meltano.yml` that get automatically set to `dev`, you can list and change the active environment using:
```

Comment:
Do I understand correctly that this PR doesn't impact how `default_environment` does or does not get applied? For instance, `default_environment` would today be ignored for `meltano config` but respected for `meltano run`.

My observation is that this part looks unchanged in the PR (which is a good thing for preserving those behaviors) and I just want to confirm that expectation is correct.

@WillDaSilva - Do I understand that correctly?

Reply:
@aaronsteers That is correct. This PR has no impact on the application of `default_environment`.

Reply:
Great. Thanks for confirming.